spec/menu_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'menu'
+
+describe Menu do
+
+  let(:pasta) {double(:item, name: "Pasta", price: 5)}
+  let(:pizza) { double(:item, name: "Pizza", price: 7) }
+
+  subject {described_class.new()}
+
+
+  it 'can return list of items and prices' do
+    expect(subject.check).to eq subject.items
```

Comment:
This test is fairly vacuous, it just confirms that `check` and `items` return the same thing: which could be something, or could be`nil`.